### ISSUE TITLE
Getting total_grad_norm as nan in the first 20 iterations

### ISSUE THREAD

**Issue by AaronC-BME** [RobinWu218/SimDINO#38]
I am not sure if this is something I should be concerned about. But for some reason when I am doing the pretraining, the total_grad_norm starts off as nan in the beginning and later on become normal values. 

![Image](https://github.com/user-attachments/assets/d745dfd7-d102-4d87-a0b5-cff54ee52d2a)

**Comment by RobinWu218** [comment]
Hi @AaronC-BME sorry for the late response. This seems weird. Which configuration are you using? And could you reproduce the results?

**Comment by AaronC-BME** [comment]
The issue no longer appear once I turn the grad_scaler to False in the configuration.
I was also wondering if you could show the training/validation loss curve during classification finetuning.

**Comment by RobinWu218** [comment]
The default config should have no such problems. We can better assist if you provide what you changed from our default settings.

What do you mean by classification fine-tuning? Please specify which particular experiment in the paper you are referring to.
